Victor-Ugwueze/MyDiary
# ##159445397 Update user profile

### What does this PR do?
This PR will add delete entry feature, with testing.

### Description of task to be completed ?
Implement the following:
 Test for 
 * Update user details endpoint
 * Get user details
 * get total number of entries endpoint
Add  profileController and profile model
Add controllers
Add verify Authentication midleware to this endpoints
The two enpoints that are 
      * `GET /api/v1/users/profile`
      * `PUT`/api/v1/users/profile` 
       * `GET /api/v1/users/profile/entries`

### How should this be tested ?

You should have PostgreSQL installed before you get started. Then clone this repo, run npm install to install all dependencies and devDependencis.
Open server/helpers/dbHelper.js change the connection string to match that of your local machine.
Then run `npm test` , and then `npm start-dev`
start POSTMAN
`POST /auth/login` , enter email as `test9@gmail.com`, password `test123`

Test the above listed endpoints
## What are the relevant pivotal tracker stories?
#159445397

### Any background context you wish to provide?
None
